### PR TITLE
Rex refactor

### PR DESCRIPTION
**Pull request by MRossol**
Migrate resource handlers and utilities to rex

### CODE REVIEW

File: reV/handlers/__init__.py [hunk 1]
```diff
@@ -3,7 +3,6 @@
 Sub-package of data handlers
 """
 from .collection import Collector
+from .exclusions import ExclusionLayers
 from .multi_year import MultiYear
-from .resource import Resource
-from .rev_resource import (NSRDB, MultiFileNSRDB, MultiFileWTK,
-                           SolarResource, WindResource)
+from .outputs import Outputs
```
Comment (grantbuster) [inline reV/handlers/__init__.py:8]: Eventually we might want to think about moving collection, multi year, and outputs to rex as well. I'm sure a lot of people in SEAC would start using those tools if they were flexible enough. For now though reV is an easy dependency too.

File: tests/peregrine.py [hunk 2]
```diff
@@ -19,8 +19,8 @@
 
 from reV import TESTDATADIR
 from reV.handlers.outputs import Outputs
-from reV.utilities.execution import PBS
-from reV.utilities.loggers import init_logger
+from rex.utilities.execution import PBS
```
Comment (grantbuster) [inline tests/peregrine.py:22]: We might just want to remove this file (peregrine.py). We're definitely not maintaining it.

File: reV/handlers/__init__.py (hunk 1 above)
Comment (MRossol) [reply reV/handlers/__init__.py:8]: I could see outputs over Collection and MY, but it still feels like more of a reV tool than a rex utility, at least we now have the option for later!

File: tests/peregrine.py (hunk 2 above)
Comment (MRossol) [reply tests/peregrine.py:22]: love it, gone!

**Pull request merged**